schema-admin.graphql

```diff
@@ -877,6 +877,11 @@ type Query {
     Retrieves all ScheduledSurfaces available to the given SSO user. Requires an Authorization header.
     """
     getScheduledSurfacesForUser: [ScheduledSurface!]!
+
+    """
+    Tool to get OG description of URL that is not yet implemented in parser
+    """
+    getOpenGraphFields(url: Url!): OpenGraphFields!
```

Comment:
We can make the return optional / nullable here. I think the benefit of that is in the client code we can just check `if(getOpenGraphField){...}` instead of `if(getOpenGraphField.description){...}` since according to the gql schema, `description` prop on that type could be `null`.

See comments below!